### PR TITLE
Add descriptions for more BP registers

### PR DESCRIPTION
**Pull request by Pokechu22**
I noticed that several registers (mostly those related to tmem) didn't have useful descriptions in the fifo analyzer while looking into https://bugs.dolphin-emu.org/issues/13377. This adds descriptions for those (and makes a few other related cleanup changes). There should be no noticeable changes to emulation behavior.

### CODE REVIEW

File: Source/Core/VideoCommon/BPStructs.cpp [hunk 1]
```diff
@@ -517,6 +517,7 @@ static void BPWritten(PixelShaderManager& pixel_shader_manager,
 
   case BPMEM_MIPMAP_STRIDE:  // MipMap Stride Channel
   case BPMEM_COPYYSCALE:     // Display Copy Y Scale
+    return;
```
Comment (Tilka) [inline Source/Core/VideoCommon/BPStructs.cpp:528]: This should really be in a separate PR.

**Review by Tilka — APPROVED** [review]

File: Source/Core/VideoCommon/BPStructs.cpp (hunk 1 above)
Comment (Pokechu22) [reply Source/Core/VideoCommon/BPStructs.cpp]: Moved to #12288.
